src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -94,15 +95,16 @@
~                        ExecutionException = ex;
~                        return FAILURE;
~                    }
                     catch (Exception ex)
```

Comment:
>LogUnhandledEngineException [](http://example.com/codeflow?start=48&length=27)

This helper previously returned an error condition to OR into RuntimeErrors. I've updated the helper to be consistent with other utility methods. We now retrieve the RuntimeErrors in the finally block.